---
http_interactions:
- request:
    method: post
    uri: https://cognito-idp.us-east-1.amazonaws.com/
    body:
      encoding: UTF-8
      string: '{"ClientId":"76jcq337aei27va9q9gtaep0dh","AuthFlow":"USER_PASSWORD_AUTH","AuthParameters":{"USERNAME":"jack@black.com","PASSWORD":"changeME123!"}}'
    headers:
      Content-Type:
      - application/x-amz-json-1.1
      Accept-Encoding:
      - ''
      User-Agent:
      - aws-sdk-ruby3/3.38.0 ruby/2.5.3 x86_64-darwin18 aws-sdk-cognitoidentityprovider/1.10.0
      X-Amz-Target:
      - AWSCognitoIdentityProviderService.InitiateAuth
      Host:
      - cognito-idp.us-east-1.amazonaws.com
      X-Amz-Date:
      - 20181116T190235Z
      X-Amz-Content-Sha256:
      - c5102059c3a24460e89bc2f43d62ca67b7394b43a063412bdb32fe354d466765
      Authorization:
      - AWS4-HMAC-SHA256 Credential=AKIAJZGHKIZ5E5OPMZBQ/20181116/us-east-1/cognito-idp/aws4_request,
        SignedHeaders=content-type;host;x-amz-content-sha256;x-amz-date;x-amz-target,
        Signature=f744e310747a966f9a5893ac3655a663140ecee08aeac9282cd86e1d4ed2b303
      Content-Length:
      - '146'
      Accept:
      - "*/*"
  response:
    status:
      code: 400
      message: Bad Request
    headers:
      Date:
      - Fri, 16 Nov 2018 19:02:35 GMT
      Content-Type:
      - application/x-amz-json-1.1
      Content-Length:
      - '67'
      Connection:
      - keep-alive
      X-Amzn-Requestid:
      - 2d92813e-e9d2-11e8-a25b-25773144c8c5
      X-Amzn-Errortype:
      - 'UserNotFoundException:'
      X-Amzn-Errormessage:
      - User does not exist.
    body:
      encoding: UTF-8
      string: '{"__type":"UserNotFoundException","message":"User does not exist."}'
    http_version: 
  recorded_at: Fri, 16 Nov 2018 19:02:35 GMT
recorded_with: VCR 4.0.0
